src/app/core/services/http/http.service.ts

```diff
@@ -38,14 +40,22 @@ export class HttpService {
   }
 
   async setHeaders() {
+    const browser = Bowser.getParser(window.navigator.userAgent);
     let token = await this.getToken();
     const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
     const acceptLanguage = await this.localStorage.getLocalData(localKeys.SELECTED_LANGUAGE);
+    const metaData = {
```

Comment:
is this needed here?

Reply:
i didnt make any change here its already there.